r/vignettes/developers/install_details.Rmd

```diff
@@ -0,0 +1,123 @@
+# How the R package is installed
+
+In order for the arrow R package to work, it needs the Arrow C++ library, 
+also known as libarrow.  There are a number of scripts that are triggered 
+when `R CMD INSTALL .` is run and for Arrow users, these should all just work 
+without configuration and pull in the most complete pieces (e.g. official 
+binaries that we host).
+
+This documented is intended specifically for arrow _developers_ who wish to know 
+more about these scripts.  If you are an arrow _user_ looking for help with 
+installing arrow, please see [the installation guide](../install.html)
+
+An overview of these scripts is shown below:
+
+* `configure` and `configure.win` - these scripts are triggered during
+`R CMD INSTALL .` on non-Windows and Windows platforms, respectively. They
+handle finding the libarrow, setting up the build variables necessary, and
+writing the package Makevars file that is used to compile the C++ code in the R
+package.
+
+* `tools/nixlibs.R` - this script is sometimes called by `configure` on Linux
+(or on any non-windows OS with the environment variable
+`FORCE_BUNDLED_BUILD=true`) if an existing libarrow installation cannot be found.
+This sets up the build process for our bundled builds (which is the default on 
+linux) and checks for binaries or downloads libarrow from source depending on 
+dependency availability and build configuration.
+
+* `tools/winlibs.R` - this script is sometimes called by `configure.win` on Windows
+when environment variable `ARROW_HOME=true`.  It looks for an existing libarrow
```

Comment:
This isn't quite right. `configure.win` uses `tools/winlibs.R` to find / download libarrow if **`ARROW_HOME` is not set**; otherwise it uses the static libraries in `$ARROW_HOME/bin`. In other words, if you have ARROW_HOME set in your environment then we assume you've built locally and want to use that.